pkg/disttask/importinto/scheduler.go

```diff
@@ -403,13 +422,21 @@ func (sch *ImportSchedulerExt) updateCurrentTask(task *proto.Task) {
 
 type importScheduler struct {
 	*scheduler.BaseScheduler
+	store kv.Storage
 }
 
-func newImportScheduler(ctx context.Context, task *proto.Task, param scheduler.Param) scheduler.Scheduler {
+// NewImportScheduler creates a new import scheduler.
+func NewImportScheduler(
+	ctx context.Context,
+	task *proto.Task,
+	param scheduler.Param,
+	store kv.Storage,
```

Comment:
we can move it to `scheduler.Param` if `ADD-INDEX` need it later?

Reply:
ADD INDEX will get the store from `ddl` object.